### ISSUE TITLE
Simple Python example fails

### ISSUE THREAD

**Issue by tomasruizt** [xylo/intellij-postfix-templates#86]
Hi,
I love how postfix completion works in Java and would really like to have it in Python as well, but it currently fails on a simple example.
My system: Ubuntu 18.04 LTS, running IntelliJ Ultimate 2018.2.4, Build #IU 182.4505.22

Here is my templates file `tomas.postfixTemplates`:
```
.foo : bar
    ANY  →  hello
```
When I press Enter in the editor to complete the template, it just removes the postfix. Then a IntelliJ reports a fatal Error on the lower right. Any idea what this could be? Here is the stacktrace. 

```
java.lang.NullPointerException
	at de.endrullis.idea.postfixtemplates.templates.SimpleStringBasedPostfixTemplate.collectExpressions(SimpleStringBasedPostfixTemplate.java:119)
	at de.endrullis.idea.postfixtemplates.templates.SimpleStringBasedPostfixTemplate$1.getNonFilteredExpressions(SimpleStringBasedPostfixTemplate.java:150)
	at com.intellij.codeInsight.template.postfix.templates.PostfixTemplateExpressionSelectorBase.getExpressions(PostfixTemplateExpressionSelectorBase.java:60)
	at de.endrullis.idea.postfixtemplates.templates.SimpleStringBasedPostfixTemplate$1.getExpressions(SimpleStringBasedPostfixTemplate.java:158)
	at com.intellij.codeInsight.template.postfix.templates.PostfixTemplateWithExpressionSelector.expand(PostfixTemplateWithExpressionSelector.java:67)
	at com.intellij.codeInsight.template.postfix.templates.PostfixLiveTemplate.expandTemplate(PostfixLiveTemplate.java:236)
	at com.intellij.codeInsight.template.postfix.templates.PostfixLiveTemplate.expandTemplate(PostfixLiveTemplate.java:145)
	at com.intellij.codeInsight.template.postfix.completion.PostfixTemplateLookupElement.expandTemplate(PostfixTemplateLookupElement.java:60)
	at com.intellij.codeInsight.template.impl.CustomLiveTemplateLookupElement.handleInsert(CustomLiveTemplateLookupElement.java:63)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.lambda$insertItem$7(CodeCompletionHandlerBase.java:547)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:1038)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.insertItem(CodeCompletionHandlerBase.java:534)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.access$000(CodeCompletionHandlerBase.java:65)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase$1.perform(CodeCompletionHandlerBase.java:468)
	at com.intellij.openapi.editor.impl.CaretModelImpl.lambda$runForEachCaret$2(CaretModelImpl.java:363)
	at com.intellij.openapi.editor.impl.CaretModelImpl.doWithCaretMerging(CaretModelImpl.java:452)
	at com.intellij.openapi.editor.impl.CaretModelImpl.runForEachCaret(CaretModelImpl.java:355)
	at com.intellij.openapi.editor.impl.CaretModelImpl.runForEachCaret(CaretModelImpl.java:346)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.insertItemHonorBlockSelection(CodeCompletionHandlerBase.java:457)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.lookupItemSelected(CodeCompletionHandlerBase.java:415)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.itemSelected(CompletionProgressIndicator.java:174)
	at com.intellij.codeInsight.completion.CompletionLookupArrangerImpl.itemSelected(CompletionLookupArrangerImpl.java:176)
	at com.intellij.codeInsight.lookup.impl.LookupImpl.fireItemSelected(LookupImpl.java:845)
	at com.intellij.codeInsight.lookup.impl.LookupImpl.finishLookupInWritableFile(LookupImpl.java:522)
	at com.intellij.codeInsight.lookup.impl.LookupImpl.lambda$finishLookup$12(LookupImpl.java:479)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:131)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:97)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:87)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:73)
	at com.intellij.codeInsight.lookup.impl.LookupImpl.finishLookup(LookupImpl.java:479)
	at com.intellij.codeInsight.lookup.impl.LookupImpl.finishLookup(LookupImpl.java:463)
	at com.intellij.codeInsight.lookup.impl.actions.ChooseItemAction$Handler.execute(ChooseItemAction.java:82)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler.doExecute(EditorActionHandler.java:173)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler.lambda$execute$4(EditorActionHandler.java:207)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler.doIfEnabled(EditorActionHandler.java:110)
	at com.intellij.openapi.editor.actionSystem.EditorActionHandler.execute(EditorActionHandler.java:206)
	at com.intellij.openapi.editor.actionSystem.EditorAction.lambda$actionPerformed$0(EditorAction.java:96)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:139)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:97)
	at com.intellij.openapi.editor.actionSystem.EditorAction.actionPerformed(EditorAction.java:105)
	at com.intellij.openapi.editor.actionSystem.EditorAction.actionPerformed(EditorAction.java:80)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:220)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:237)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:620)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$processAction$2(IdeKeyEventDispatcher.java:669)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:195)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:668)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processActionOrWaitSecondStroke(IdeKeyEventDispatcher.java:520)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:475)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:212)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:697)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by tomasruizt** [comment]
I correct myself. It failed initially, but after several restarts it is not failing anymore

**Comment by xylo** [comment]
Thanks for reporting the bug.  It's good to hear that it works for you now, but I will try to fix this NullPointerException.

**Comment by chen19901225** [comment]
I meet the same problem, have it fixed?

**Comment by xylo** [comment]
I tried to reproduce it but I failed.  It's not yet fixed.

**Comment by gejun123456** [comment]
I have fixed with #98 
you can try https://github.com/gejun123456/intellij-postfix-templates/releases to check.

**Comment by xylo** [comment]
Thanks for the fix, @gejun123456.  I just merged your code and will publish the plugin once I figured out how to deploy it with gradle. :D

**Comment by xylo** [comment]
I finally released version 2.8.2.  Sorry that it took me so long.
